# HG changeset patch
# Parent 65d595ef9fc3cdbbde4894d927593f1b0f5bbcb7
A fix for libgomp to correctly request a POSIX version for time support.

diff -r 65d595ef9fc3 libgomp/config/posix/time.c
--- a/libgomp/config/posix/time.c	Tue May 20 11:05:45 2014 -0400
+++ b/libgomp/config/posix/time.c	Tue May 20 11:05:48 2014 -0400
@@ -28,6 +28,8 @@
    The following implementation uses the most simple POSIX routines.
    If present, POSIX 4 clocks should be used instead.  */
 
+#define _POSIX_C_SOURCE 199309L /* for clocks */
+
 #include "libgomp.h"
 #include <unistd.h>
 #if TIME_WITH_SYS_TIME
